tests/v2/SharedTemplates/templates/PrimaryActorName/Repository.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\BuphaloTemplateTree\RelativeNamespace\PrimaryActorName;
+
+use Neighborhoods\BuphaloTemplateTree\RelativeNamespace\PrimaryActorName;
+use Neighborhoods\BuphaloTemplateTree\RelativeNamespace\PrimaryActorNameInterface;
+
+class Repository implements RepositoryInterface
+{
+    use PrimaryActorName\Builder\Factory\AwareTrait;
+
+    public function add(PrimaryActorNameInterface $PrimaryActorName): RepositoryInterface
+    {
+        /** @neighborhoods-buphalo:annotation-processor \Neighborhoods\BuphaloTemplateTree\RelativeNamespace\PrimaryActorName\Repository.add1
```

Comment:
I don't think it's necessary so much as a copy/paste holdover from the early days. As far as I can remember, there isn't any necessary uniqueness as annotations are only done per-template, and even then, they don't have to be unique, but if they aren't, then they'll get replaced by the same thing (which is sometimes desirable).

It's possible it was put there as a future-proof in case things like global-scope annotation tags were introduced.